### PR TITLE
Adds numArguments and  isInlined for FunctionDecl

### PR DESCRIPTION
**Pull request by achrafmam2**

### CODE REVIEW

File: Sources/Clang/Cursors.swift [hunk 1]
```diff
@@ -13,6 +13,11 @@ public struct FunctionDecl: ClangCursorBacked {
   init(clang: CXCursor) {
     self.clang = clang
   }
+
+  /// Retrieve the number of arguments of the function.
+  public var numArguments: Int {
```
Comment (CodaFi) [inline Sources/Clang/Cursors.swift]: Nit: Perhaps a better name here.  `argumentCount`?

**Comment by CodaFi** [comment]
Just a single nit.  Feel free to weigh in and replace this with whatever you prefer.  My only request is that it not use abbreviations like `num`.  C++isms cannot be allowed to breed.

File: Sources/Clang/Cursors.swift (hunk 1 above)
Comment (achrafmam2) [reply Sources/Clang/Cursors.swift]: I think that's a better name. Thanks.

File: Sources/Clang/Cursors.swift [hunk 2]
```diff
@@ -25,6 +30,11 @@ public struct FunctionDecl: ClangCursorBacked {
   public var resultType: CType? {
     return convertType(clang_getCursorResultType(clang))
   }
+
+  /// Tells if the function delclaration is inlined.
```
Comment (CodaFi) [inline Sources/Clang/Cursors.swift]: Nit: `declaration`

Comment (achrafmam2) [reply Sources/Clang/Cursors.swift]: Thanks!

**Comment by CodaFi** [comment]
⛵️

**Pull request merged**